src/content/docs/en/guides/server-side-rendering.mdx

```diff
@@ -129,18 +129,6 @@ export default defineConfig({
 });
 ```
 
-You can then choose to override the default on-demand rendering of `server` mode on any page or route by exporting `const prerender = true`:
-
-```astro title="src/pages/my-static-page.astro" ins={2}
----
-export const prerender = true;
-// ...
----
-<html>
-  <!-- Static, pre-rendered page here... -->
-</html>
-```
```

Comment:
The method for opting into pre-rendering in `server` mode is described immediately below, and it seems to be redundant. So, I think this part can be safely removed.